Linker error in Debug build

I get the error 

```
/usr/bin/ld: CMakeFiles/tixi_c_examples.dir/tixi_c_examples.c.o: undefined reference to symbol 'ceil@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
examples/Demo/CMakeFiles/tixi_c_examples.dir/build.make:98: recipe for target 'examples/Demo/tixi_c_examples' failed
```

when trying to build in Debug mode on Linux. This is fixed by explicitly linking the math library to `tixi_c_examples` in https://github.com/DLR-SC/tixi/blob/master/examples/Demo/CMakeLists.txt as its already done for the `tixiDemo` target.

--- a/examples/Demo/CMakeLists.txt
+++ b/examples/Demo/CMakeLists.txt
@@ -33,6 +33,11 @@ endif(UNIX)
 add_executable(tixi_c_examples tixi_c_examples.c)
 target_link_libraries (tixi_c_examples tixi3)
 
+if (UNIX)
+  # link math library
+  target_link_libraries (tixi_c_examples m)
+endif(UNIX)
+
 if(WIN32)
     target_compile_definitions(tixi_c_examples PRIVATE _CRT_SECURE_NO_WARNINGS)
 endif(WIN32)

